styles/style.css

```diff
@@ -316,6 +324,11 @@ td.amount {
 	border-top-width: 1px;
 }
 
+.create-form input {
+	padding-right: 78px;
+	text-overflow: ellipsis;
+}
+
```

Comment:
Not able to find out what this is doing.

Reply:
Ellipsizes when field is _not_ focused. :D